.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -1,52 +1,78 @@
 ---
-name: Cherry pick template
-about: Used to request a cherry pick. See bit.ly/amp-cherry-pick
-title: "\U0001F338 Cherry pick request for <YOUR_ISSUE_NUMBER> into <RELEASE_ISSUE_NUMBER>
-  (Pending)"
+name: Cherry-pick template
+about: Used to request a cherry-pick. See bit.ly/amp-cherry-pick
+title: "\U0001F338 Cherry-pick request for #<ISSUE_NUMBER> into #<RELEASE_ISSUE> (Pending)"
 labels: 'Type: Release'
 assignees: cramforce
-
 ---
 
 <!--
-Replace *everything* in angle brackets in the title AND body of this issue. If you have any questions see the [cherry pick documentation](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#cherry-picks).
--->
+MUST: Replace *everything* in angle brackets in the title AND body of this issue.
 
+If you have any questions see the [cherry-pick documentation](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#cherry-picks).
+-->
+# Cherry-pick request
 
-## GitHub issue your cherry pick is fixing:
+<!--
+TIP: Cherry-picks into production most likely require a cherry-pick into RC too. Otherwise, your fix will be lost when the RC is promoted.
+-->
 
-Issue #<ISSUE_NUMBER>
+| Issue | PR  | Production? | RC? | [Release issue](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) |
+| :---: | :-: | :---------: | :-: | :-----------: |
+| #<_ISSUE_NUMBER_> | #<_PR_NUMBER_> | **<YES/NO>** | **<YES/NO>** | #<_RELEASE_ISSUE_> |
 
+## Why does this issue meet the [cherry-pick criteria](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#cherry-pick-criteria)?
 
-## PR that you are requesting a cherry pick for:
+<!--
+TIP: Be specific.
+-->
+<_YOUR_REASONS_>
 
 <!--
-Put N/A if you do not yet have a PR with a fix; edit this issue to add it when the PR is ready.
+CONDITION: Cherry-picking into production but _not_ RC. Otherwise, delete.
 -->
+## Why is a RC cherry-pick not needed?
 
-PR #<PR_NUMBER>
+<_YOUR_REASONS_>
 
+<!--
+MUST: Filling out the mini-PM template is required _after_ the deployment of a production cherry-pick. If this cherry-pick does not include production, the mini-PM section can be deleted.
 
-## Release(s) you requesting this cherry pick into:
+MUST: This issue cannot be closed until the mini-PM is written and its action items are completed.
+-->
+# Mini-postmortem
```

Comment:
Someone viewing the issue will see a bunch of blank sections here, which may be confusing.  Perhaps:

- add a note visible in the issue indicating "Postmortem will be completed before this issue is closed."
- or put a comment block around these sections so they're not visible in the issue initially, with instructions to make the post-mortem visible when it is ready

Reply:
Good idea, done.